apps/verticalhome/js/more_apps_navigation.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+(function (exports) {
+
+  var MoreAppsNavigation = {
+    controls: null,
+    selectedElemIndex: 0,
+    selectElement: null,
+
+    init: function () {
+      this.controls = app.grid.getItems();
+      this.reset();
```

Comment:
Strange to call reset() after setting the controls. I have to go check whether that resets the controls array.